modules/new_serverpod_auth/serverpod_auth_session/serverpod_auth_session_server/lib/src/business/auth_sessions.dart

```diff
@@ -138,16 +164,29 @@ abstract final class AuthSessions {
       session,
       AuthSession(
         authUserId: authUserId,
+        created: clock.now(),
```

Comment:
@SandPod Would you be up for using `clock` inside the models? I think that'd be a boon, as those implicit "now" dates are easily forgotten to make testable :)